sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -148,8 +171,15 @@ case class Hour(child: Expression) extends UnaryExpression with ImplicitCastInpu
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(param) - Returns the minute component of the string/timestamp/interval.",
-  extended = "> SELECT _FUNC_('2009-07-30 12:58:59');\n 58")
+  usage = "_FUNC_(timestamp) - Returns the minute component of the string/timestamp.",
+  extended = """
+    Arguments:
+      timestamp - a timestamp expression.
```

Comment:
Oh, yes. I knew this but I was hesitant to fix `string/timestamp` because fixing this to `timestamp` sounds a (strictly..) regression in the documentation (it mean, `string/timestamp` is not inaccurate..); however, I also could not add `string` in `extended` part as it is inconsistent with the others.